templates/download/shared/_first-boot-setup.html

```diff
@@ -8,7 +8,7 @@ <h2 class="p-stepped-list__title">First boot setup</h2>
       <li class="p-list__item">Press enter then select "Start" to begin configuring your network and an administrator account. Follow the instructions on the screen, you will be asked to configure your network and enter your Ubuntu SSO credentials.</li>
       <li class="p-list__item">
         At the end of the process, you will see your credentials to access your Ubuntu Core machine:
-        <p></p>
+        <br /><br />
         <pre><code>This device is registered to &lt;Ubuntu SSO email address&gt;.
```

Comment:
```suggestion
        <pre style="margin-top: 1rem"><code>This device is registered to &lt;Ubuntu SSO email address&gt;.
```
I believe it's best practice to use margin for spacing instead of brs